scripts/conversion/convert_diffusers_autoencoder_kl.py

```diff
@@ -19,7 +19,7 @@ class Args(argparse.Namespace):
 def setup_converter(args: Args) -> ModelConverter:
     target = LatentDiffusionAutoencoder()
     source: nn.Module = AutoencoderKL.from_pretrained(  # type: ignore
-        pretrained_model_name_or_path=args.source_path, subfolder=args.subfolder
+        pretrained_model_name_or_path=args.source_path, subfolder=args.subfolder, low_cpu_mem_usage=False
```

Comment:
This just removes a lot of annoying messages from the console asking us to `pip install accelerate`

Reply:
Good idea, but maybe add a comment in the script to explain that.

Reply:
Interesting, I would have thought the auto formatter made that choice for me. TIL

Reply:
Done